data/pathfinder/paizo/campaign_setting/faction_guide/pcfg_equipmods.lst

```diff
@@ -15,10 +15,10 @@ Nexavaran Steel								KEY:Material ~ Nexavaran Steel			NAMEISPI:YES	TYPE:Weapon
-Jarring									KEY:Special Quality ~ Jarring ~ Armor					TYPE:Armor			COST:1000			VISIBLE:QUALIFY	SOURCEPAGE:p.55	PRETYPE:1,Armor			BONUS:MOVEADD|TYPE.Walk|-5|!PREABILITY:1,CATEGORY=Special Ability,TYPE.DwarfSteady,TYPE.DuergarSteady	BONUS:EQM|WEIGHTADD|20	BONUS:ITEMCOST|TYPE.Enhancement.WEAPON|500	SPROP:If attacked by Power Attack with a melee weapon that misses by 5 or less, attacker must make a Fortitude (DC 10 plus bonus Power Attack would have applied to damage).
-Vital Guard									KEY:Special Quality ~ Vital Guard ~ Armor					TYPE:Armor			COST:500			VISIBLE:QUALIFY	SOURCEPAGE:p.55	PRETYPE:1,Armor			BONUS:MOVEADD|TYPE.Walk|-5|!PREABILITY:1,CATEGORY=Special Ability,TYPE.DwarfSteady,TYPE.DuergarSteady	BONUS:EQM|WEIGHTADD|15	BONUS:ITEMCOST|TYPE.Enhancement.WEAPON|250	SPROP:Treat the wearer's armor bonus as +2 higher than normal for the confirmation roll against critical threats.
+Deflecting									KEY:Special Quality ~ Deflecting ~ Armor					TYPE:Armor			COST:1000			VISIBLE:QUALIFY	SOURCEPAGE:p.55	PRETYPE:2,ArmorEnhancement,Armor			BONUS:MOVEADD|TYPE.Walk|-5|!PREABILITY:1,CATEGORY=Special Ability,TYPE.DwarfSteady,TYPE.DuergarSteady	BONUS:EQM|WEIGHTADD|10	BONUS:ITEMCOST|TYPE.Enhancement.WEAPON|500	SPROP:If attacked by a bludgeoning or slashing melee weapon that misses by 5 or less, wearer gains +1 on his next melee attack roll against that opponent if made in the next round.
+Drowscourge									KEY:Special Quality ~ Drowscourge ~ Weapon				TYPE:Weapon	PLUS:1					VISIBLE:QUALIFY	SOURCEPAGE:p.59	PRETYPE:2,WeaponEnhancement,Weapon																														SPROP:Against driders, creatures of drow subtype, the weapon's enhancement bonus is +1 better. The bearer is able to detect drow 1/day and gain darkvision (as the spell) 1/day.
+Jarring									KEY:Special Quality ~ Jarring ~ Armor					TYPE:Armor			COST:1000			VISIBLE:QUALIFY	SOURCEPAGE:p.55	PRETYPE:2,ArmorEnhancement,Armor			BONUS:MOVEADD|TYPE.Walk|-5|!PREABILITY:1,CATEGORY=Special Ability,TYPE.DwarfSteady,TYPE.DuergarSteady	BONUS:EQM|WEIGHTADD|20	BONUS:ITEMCOST|TYPE.Enhancement.WEAPON|500	SPROP:If attacked by Power Attack with a melee weapon that misses by 5 or less, attacker must make a Fortitude (DC 10 plus bonus Power Attack would have applied to damage).
+Vital Guard									KEY:Special Quality ~ Vital Guard ~ Armor					TYPE:Armor			COST:500			VISIBLE:QUALIFY	SOURCEPAGE:p.55	PRETYPE:2,ArmorEnhancement,Armor			BONUS:MOVEADD|TYPE.Walk|-5|!PREABILITY:1,CATEGORY=Special Ability,TYPE.DwarfSteady,TYPE.DuergarSteady	BONUS:EQM|WEIGHTADD|15	BONUS:ITEMCOST|TYPE.Enhancement.WEAPON|250	SPROP:Treat the wearer's armor bonus as +2 higher than normal for the confirmation roll against critical threats.
```

Comment:
Deflecting, Jarring, and Vital Guard are mundane modification and don't require magic armor (no need for ArmorEnhancement in the PRETYPE).